graphdatascience/graph_data_science.py

```diff
@@ -72,9 +99,7 @@ def driver_config(self) -> Dict[str, Any]:
         return self._config
 
     @classmethod
-    def from_neo4j_driver(cls: Type[GDS], driver: Driver) -> "GraphDataScience":
-        return cls(cls.create_neo4j_query_runner(driver))
-
-    @staticmethod
-    def create_neo4j_query_runner(driver: Driver, auto_close: bool = False) -> Neo4jQueryRunner:
-        return Neo4jQueryRunner(driver, auto_close=auto_close)
+    def from_neo4j_driver(
+        cls: Type[GDS], driver: Driver, auth: Optional[Tuple[str, str]] = None, arrow: bool = True
+    ) -> "GraphDataScience":
```

Comment:
We cannot reference `GraphDataScience` directly since it's not fully defined at this point, so we have to use a string instead - that's Python for you :man_shrugging:

Reply:
Redefining stringly-typed, I see 🙈